Extension/src/Debugger/extension.ts

```diff
@@ -21,7 +21,11 @@ const localize: nls.LocalizeFunc = nls.loadMessageBundle();
 const disposables: vscode.Disposable[] = [];
 
 export function buildAndDebugActiveFileStr(): string {
-    return ` - ${localize("build.and.debug.active.file", 'Build and debug active file')}`;
+    return `${localize("build.and.debug.active.file", 'Build and debug active file')}`;
+}
+
+export function CppExtensionSourceStr(): string {
+    return `${localize("Cpp.extension.source", 'C/C++')}`;
```

Comment:
This does not need to be localized and since it's a constant string, you can just define a const string inside of `configurationProvider.ts` instead of making this a function. It doesn't look like this is used anywhere else.